README.md

```diff
@@ -172,6 +172,25 @@ For example:
 ┣ ┣ ┃ ┃ ┣ 📜 UserProfile.stories.mdx \
 ```
 
+#### Scoped Files 
+
+We need to add inside `pages/**/{utils, helpers, context, hooks, etc...}` and use `camelCase` as **Naming Convention**.
+
+If we need to used this files again in another context or page it need to be moved to `src/{utils, helpers, context, hooks}`.
```

Comment:
```suggestion
If we need to use these files again in another context or page it need to be moved to `src/{utils, helpers, context, hooks}`.
```

Reply:
thanks!! fix in [91eaf3b](https://github.com/juntossomosmais/frontend-guideline/pull/29/commits/91eaf3b5bc124ec841b837a8113ac43c04012258)